docs/src/pages/docs/en/internationalization/adding-language-support.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Adding Language Support
+description: Adding Language Support
+layout: ../../../../layouts/MainLayout.astro
+---
+
+Media Chrome supports multiple languages for its UI components. By default, it uses English, but you can include additional languages by loading the corresponding language module.
+
+## Available Languages
+Currently, the supported languages are:
+
+- **English** (default)
+- **Spanish** (`es`)
+- **French** (`fr`)
+- **Portuguese** (`pt`)
+
+## How to Include a Language
+
+### Using HTML
+To add support for a specific language in an HTML file, include the corresponding language module using a `<script>` tag:
+
+```html
+<script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome@4.8.0/dist/lang/pt.js/+esm"></script>
+```
+
+Replace `pt.js` with the appropriate language code (`es.js`, `fr.js`, etc.) for other languages.
+
+### Using JavaScript
+If you are working with JavaScript, you can import the language module as follows:
+
+```js
+import "media-chrome/dist/lang/pt.js";
```

Comment:
```suggestion
import "media-chrome/lang/pt.js";
```

remove the dist part if possible. I noticed we might need to add one more exports block before the other one to make this work. this one

```
    "./lang/*.js": {
      "import": "./dist/lang/*.js",
      "require": "./dist/lang/*.js",
      "default": "./dist/lang/*.js"
    }
```